### PR TITLE
Add an ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by fkaelberer**
Github recently introduced issue templates that help contributors to provide necessary information.
I think this should reduce the number of information requests. What do you think about such a template?

https://github.com/blog/2111-issue-and-pull-request-templates

Edit: Try the template at https://github.com/fkaelberer/pdf.js/issues/new, resulting issues look like https://github.com/fkaelberer/pdf.js/issues/4.

### CODE REVIEW

File: ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#### Expected behavior
+
+#### Actual behavior
+
+#### Steps to reproduce the behavior
+
+---
+##### Link to PDF file
```
Comment (Snuffleupagus) [inline ISSUE_TEMPLATE]: Perhaps it would be a good idea to also indicate that GitHub now supports attaching PDF files to issues?

**Comment by Snuffleupagus** [comment]
I think this is a great idea!
Besides my comment above, would it make sense to *also* add a link to https://github.com/mozilla/pdf.js/blob/master/CONTRIBUTING.md at the top of the template (e.g. "Please also read CONTRIBUTING.md", or something like that)?

**Comment by yurydelendik** [comment]
> "Please also read CONTRIBUTING.md"

We can keep this out. However I would move both ISSUE_TEMPLATE.md and CONTRIBUTING.md into .github/ folder (as post suggested)

**Comment by yurydelendik** [comment]
I was thinking about something less busy, e.g.:

```
Link to PDF file (or attach file here):

Steps to reproduce the problem:

What is the expected behavior?

What went wrong?

Configuration:
- Web browser and its version:
- Operating system and its version:
- PDF.js version:
- Is an extension:

Link to a viewer (if hosted on the site other than mozilla.github.io or as an extension):

```

**Comment by timvandermeij** [comment]
I definitely think we should add this. I like @yurydelendik's suggestion as it's more compact, yet still contains all required information.

**Comment by fkaelberer** [comment]
Thanks for your feedback. @yurydelendik, I used your text.

**Comment by timvandermeij** [comment]
Looks good, thank you! I'm confident that this will lead to better issue descriptions.

**Pull request merged**